Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gradle:6.7.1-jdk8-openj9 AS gradle-build
```

Comment:
Why are we changing this?

Reply:
The previous version is not working with M1 chip macbook, I needed to upgrade this to make it work on my Mac but if we don't need it, we can keep the previous version

Reply:
cool, there was no reasoning in the commit message for this change.
grade update from 6.7.1 to 6.9.1

In the future it would be great to have a separate logical commit to explain the change so that history lookup is easier.